src/hooks/useContent.js

```diff
@@ -124,11 +149,5 @@ const useContent = ({
   }
 }
 
-useContent.defaultProps = {
```

Comment:
Why not keep using defaultProps?

Reply:
I discovered in testing that they were not working.  That's when I realized that for functions you specify defaults in the parameter list, so I moved them there.  Functions don't have props, they have parameters.

Reply:
Good catch, I thought hooks behave like components with default props. Good to know they actually don't